private-distributed-v2.yml

```diff
@@ -1,14 +1,23 @@
 # Allow list for closed-source distributed projects.
 allow-licenses:
   - '0BSD'
+  - '0BSD AND ISC AND MIT'
```

Comment:
The latest version doesn't support spdx expression in allow list, a lib with `0BSD AND ISC AND MIT` will be allowed by
```
- 0BSD
- ISC
- MIT